src/components/CollaborationBar/CollaborationBar.test.tsx

```diff
@@ -103,4 +104,55 @@ describe('<CollaborationBar>', () => {
       }),
     ).toBeInTheDocument();
   });
+
+  it('should show cursors in presentation mode if edit mode is enabled', async () => {
+    setPresentationMode(true, true);
+
+    render(<CollaborationBar />, { wrapper: Wrapper });
+
+    await userEvent.click(
+      screen.getByRole('checkbox', {
+        name: "Show collaborators' cursors",
+        checked: false,
+      }),
+    );
+
+    expect(
+      screen.getByRole('checkbox', {
+        name: "Hide collaborators' cursors",
+        checked: true,
+      }),
+    ).toBeInTheDocument();
+  });
+
+  it('should hide cursors in presentation mode if edit mode is not enabled', async () => {
+    render(<CollaborationBar />, { wrapper: Wrapper });
+
+    const ShowCollaboratorsCursors = screen.getByRole('checkbox', {
+      name: "Show collaborators' cursors",
+      checked: false,
+    });
+
+    setPresentationMode(true, false);
+
+    expect(ShowCollaboratorsCursors).not.toBeInTheDocument();
+
+    setPresentationMode(true, true);
+
+    await userEvent.click(
+      screen.getByRole('checkbox', {
+        name: "Show collaborators' cursors",
+        checked: false,
+      }),
+    );
+
+    const hideCollaboratorsCursors = screen.getByRole('checkbox', {
+      name: "Hide collaborators' cursors",
+      checked: true,
+    });
+
+    setPresentationMode(true, false);
+
+    expect(hideCollaboratorsCursors).not.toBeInTheDocument();
```

Comment:
we could have something more simple, like:
```suggestion
    setPresentationMode(true, false);

    render(<CollaborationBar />, { wrapper: Wrapper });

    expect(
      screen.queryByRole('checkbox', { name: /cursors$/ }),
    ).not.toBeInTheDocument();
```